forui/lib/src/widgets/pagination/pagination.dart

```diff
@@ -0,0 +1,216 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'package:meta/meta.dart';
+
+import 'package:forui/forui.dart';
+import 'package:forui/src/widgets/pagination/pagination_controller.dart';
+
+/// A pagination enables the user to select a specific page from a range of pages.
+///
+/// See:
+/// * https://forui.dev/docs/navigation/pagination for working examples.
+/// * [FPaginationController] for customizing the pagination component's behavior.
+/// * [FPaginationStyle] for customizing the pagination component's appearance.
+final class FPagination extends StatefulWidget {
+  /// The pagination's style. Defaults to the appropriate style in [FThemeData.paginationStyle].
```

Comment:
According to this documentation, paginationStyle contains multiple styles. Pls fix.